ppl/src/main/java/com/amazon/opendistroforelasticsearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -197,6 +206,56 @@ public UnresolvedPlan visitEvalCommand(EvalCommandContext ctx) {
     );
   }
 
+  private List<UnresolvedExpression> getGroupByList(ByClauseContext ctx) {
```

Comment:
could you also change the stats command to use these genric method?

Reply:
will do

Reply:
used getGroupByList in stats command